### DISCUSSION TITLE
CMD + Enter

### DISCUSSION THREAD

**Discussion by chrisrowe** [trendyminds/craft-palette#9] [Q&A] [answered]
Not sure if kbar supports this but I would also love the ability to open results in a new tab, maybe with CMD + Enter

**Comment by aaronbushnell** [comment]
Hey @chrisrowe — unfortunately kbar doesn't have this type of functionality baked in at this point. If they include that capability I'll whip up a point release to add that option!

**Reply by chrisrowe** [reply]
Hey, thanks so much for checking!

**Comment by wmdhosting** [comment]
yes this wil be great.. or dont know if posiblle in Craft 4  to open  result in slideout...  that will solve workflow too . even better..

**Comment by wmdhosting** [comment]
https://dev-diary.newism.com.au/posts/creating-craft-cms-slideouts.html

**Comment by aaronbushnell** [comment, marked as answer]
Palette 3.0 now supports opening links in a new tab using either `CMD+Enter` or `CMD+click`!
